### DISCUSSION TITLE
Allow Weighting up to 3 in Flux LoRAs

### DISCUSSION THREAD

**Discussion by DavidDragonsage** [runew0lf/RuinedFooocus#202] [Ideas]
The Anti-blur Flux Lora
https://civitai.com/models/675581/anti-blur-flux-lora
works well with weightings up to a value of 3. In fact it really gains its full potential for extending depth of field when set to this value.

There currently is a workaround available in RF, you can load the metadata for a previously created image and edit the weighting for this LoRA, then regenerate. But this is rather awkward. If a maximum LoRA weighting of 3 could be set directly from the interface, as it is in mainline Fooocus, that would be great!

**Comment by yownas** [comment]
I've seen a request like this before. We decided to have the range 0-2 simply because it made more sense. Having a bigger range made the slider basically useless. But... I added so you can set the range in `settings\settings.json`

```
  "lora_min": -1,
  "lora_max": 4
}
```

(Just be careful with the `,`. json is a bit silly, all items except the last should have one.)

**Reply by DavidDragonsage** [reply]
Excellent, thank you! 🙂
